pkg/obfuscate/appsec.go

```diff
@@ -0,0 +1,538 @@
+// Unless explicitly stated otherwise all files in this repository are licensed
+// under the Apache License Version 2.0.
+// This product includes software developed at Datadog (https://www.datadoghq.com/).
+// Copyright 2016-present Datadog, Inc.
+
+package obfuscate
+
+import (
+	"fmt"
+	"regexp"
+	"sort"
+	"strings"
+)
+
+// ObfuscateAppSec obfuscates the given appsec tag value in order to remove
+// sensitive values from the appsec security events.
+func (o *Obfuscator) ObfuscateAppSec(val string) string {
+	keyRE := o.opts.AppSec.ParameterKeyRegexp
+	valueRE := o.opts.AppSec.ParameterValueRegexp
+	if keyRE == nil && valueRE == nil {
+		return val
+	}
+	ob := appsecEventsObfuscator{
+		keyRE:   keyRE,
+		valueRE: valueRE,
+	}
+	output, err := ob.obfuscate(val)
+	if err != nil {
+		o.log.Errorf("unexpected error while obfuscating the appsec events: %v", err)
```

Comment:
```suggestion
		o.log.Errorf("Unexpected error while obfuscating the appsec events: %v", err)
```
Please check that all logged messages are capitalised.

Reply:
Fixed by https://github.com/DataDog/datadog-agent/pull/11324/commits/48234cfc628802b9faf4ed9289d9742eaf5dc83d